### ISSUE TITLE
[autocomplete] el-scrollbar not register

### ISSUE THREAD

**Issue by OMGVecchio** [ElemeFE/element#3021]
### ElementUI version

1.2.1

### OS/Browers version

macOS/Chrome 55.0.2883.95 (64-bit)

### Vue version

2.1.10

### Steps to reproduce
just use the el-autocomplete component like this
![image](https://cloud.githubusercontent.com/assets/12711817/23292120/6337568e-fa98-11e6-9d1f-dd0a40711a74.png)

### What is Expected?
successful result

### What is actually happening?
![image](https://cloud.githubusercontent.com/assets/12711817/23292140/810d6d06-fa98-11e6-85e2-9fbfaa5c4ca5.png)
`el-scrollbar` is not parsed

**Comment by tenCats** [comment]
same error.
element-ui: 1.2.3
vue.js: ^2.1.0

**Comment by luxp** [comment]
have the same issue with v1.2.4
I fixed this by registering the scrollbar component to the global 
```javascript
import {Scrollbar} from 'element-ui'
Vue.component(Scrollbar.name, Scrollbar
```
but I think it's not the prefect way.

**Comment by Leopoldthecoder** [comment]
Fixed in #3474
